modules/ROOT/pages/dataweave-system-properties.adoc

```diff
@@ -199,44 +357,108 @@ output application/json
 }
 ----
 
-Available for language levels: 2.3, 2.4
 
-//com.mulesoft.dw.xml_reader.honourMixedContentStructure//
-|`com.mulesoft.dw.xml_reader.honourMixedContentStructure`|
-When this property is set to `true`, DataWeave retains a mixed-content structure instead of grouping text with mixed content into a single text field.
+* Type: `Boolean`
+
+* Default: `
+  false for language levels: 2.3, 2.4
+
+  false for language levels: 2.5`
+
+Available for language levels: 2.3, 2.4, 2.5
+
+
+//com.mulesoft.dw.workingdirectory.base//
+|`com.mulesoft.dw.workingdirectory.base`|
+Specifies the base directory in which temporary files will be written
+
+* Type: `String`
+* Default: `java.io.tmpdir (system property value)`
+
+
+//com.mulesoft.dw.workingdirectory.delete_sync//
+|`com.mulesoft.dw.workingdirectory.delete_sync`|
+File deletion occurs synchronously when this property is set to `true`.
 
 * Type: `Boolean`
 * Default: `false`
 
-Available for language levels: 2.4
 
-//com.mulesoft.dw.xml_reader.parseDtd//
-|`com.mulesoft.dw.xml_reader.parseDtd`|
-DataWeave parses a Doctype declaration when this property is set to `true`.
+//com.mulesoft.dw.workingdirectory.max_output_buffer_size//
+|`com.mulesoft.dw.workingdirectory.max_output_buffer_size`|
+Specifies the maximum size of a file to be written in bytes
+
+* Type: `Long`
+* Default: `-1`
+
+
+//com.mulesoft.dw.workingdirectory.tracking//
+|`com.mulesoft.dw.workingdirectory.tracking`|
+Each script generates temporary files in their own directory when this property is set to `true`.
 
 * Type: `Boolean`
 * Default: `false`
 
-Available for language levels: 2.5
 
 //com.mulesoft.dw.xml.supportDTD//
 |`com.mulesoft.dw.xml.supportDTD`|
-Controls whether DTD handling is enabled or disabled. When this property is set to `false`, DataWeaves skips processing of both internal and external subsets. Note that the default for this property changed from `true` to `false` in Mule version 4.3.0-20210427, which includes the May, 2021 patch of DataWeave version 2.3.0.
+Controls whether DTD handling is enabled or disabled. When this property is set to `false`, DataWeave skips processing of both internal and external subsets.
 
 * Type: `Boolean`
 * Default: `false`
 
-//com.mulesoft.dw.charbuffersize//
-|`com.mulesoft.dw.charbuffersize`|
-Determines the size (in bytes) of the buffer used by the writer.
 
-* Type: `Number`
-* Default: `4096` 
+//com.mulesoft.dw.xml_reader.honourMixedContentStructure//
+|`com.mulesoft.dw.xml_reader.honourMixedContentStructure`|
 
-//mule.weave.script.parser.cache.size//
-|`mule.weave.script.parser.cache.size`|
-Determines the maximum size of compiled scripts that are being cached..
+When this property is set to `true`, DataWeave retains a mixed-content structure instead of grouping text with mixed content into a single text field.
```

Comment:
```suggestion
When set to `true`, DataWeave retains a mixed-content structure instead of grouping text with mixed content into a single text field.
```